### ISSUE TITLE
RaspberryPI Image: $(internal:bind_ip) not showing ip adress

### ISSUE THREAD

**Issue by macflo1508** [bitfocus/companion#1006]
After installing companionPi from the newly released image, $(internal:bind_ip) is now showing 0.0.0.0 instead of the actual ip address. With my previous self made pi installation this was working fine.

**Comment by macflo1508** [comment]
After reading through the CompanionPi docs, I can see now that this is probably due to the fact that it binds the IP to 0.0.0.0 at startup. So let me rephrase this:
Can we change this to show the actual IP adress or add a $(... command that shows the actual IP of the used adapter? It was very convienient to see the IP adress right on the button.

**Comment by JeffreyDavidsz** [comment]
I believe this is a double request, but I feel your pain, next week I'll use my Pi I used a different script so i'll check if i can fix the difference

**Comment by haakonnessjoen** [comment]
Maybe this would be fixed better if we added variables like $(internal:ip_eth0) etc. Since it is really nice to listen to 0.0.0.0, as companion would listen on all interfaces, regardless of network changes.

**Comment by jarodwsams** [comment]
I would agree with that, @haakonnessjoen. Especially for headless it makes the most sense to listen to 0.0.0.0, so having variables for the addresses actually assigned to the interface would be helpful.

**Comment by josephdadams** [comment]
This is a duplicate of #987

**Comment by jarodwsams** [comment]
> This is a duplicate of #987

Indeed, it is. We should probably close this one in deference to 987.
